client/src/redux/storeStateType.ts

```diff
@@ -12,5 +12,6 @@ export default interface StoreStateType {
     cities: Map<string, City>;
     countries: Map<string, Country>;
     contactTypes: Map<number, ContactType>;
-    groupUsers: Map<string, User>
+    groupUsers: Map<string, User>;
+    formsValidations: (boolean | null)[];
```

Comment:
change to boolean[] due to previous comment

Reply:
watch the previous comment